### ISSUE TITLE
keycaps are blank on exported svg

### ISSUE THREAD

**Issue by 9999years** [ijprest/keyboard-layout-editor#148]
http://www.keyboard-layout-editor.com/#/gists/39ca245f8035dc1ff6d6
screenshot
![73bd1c268f89c160284d7d67b397e36c](https://cloud.githubusercontent.com/assets/15312184/10746802/7e976376-7c89-11e5-8788-6c3a268c49c1.png)

**Comment by ijprest** [comment]
Yep... SVG output is still experimental.  Text output hasn't been implemented yet.

**Comment by 9999years** [comment]
Figured I'd let you know. Really awesome tool btw, absolutely essential.
Will drag and dropping keys from one area to another be added at any point?

**Comment by ijprest** [comment]
It's on the list, but other features have been a higher priority: #2

**Comment by albertonieto** [comment]
Same issue here. Any idea when this will get resolved? I do need the SVG file, a screenshot won't work for me. Thanks!

**Comment by theArnoll** [comment]
This issue is still existing. Are there any possibility for this function to be complete?
